calico/hatch.toml

```diff
@@ -0,0 +1,8 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["38"]
+
+[envs.default.env-vars]
+TOX_ENV_NAME = "{env_name}"
```

Comment:
What's this for?

Reply:
To prevent this from failing: 
https://github.com/DataDog/integrations-core/blob/9aff7a44e3de2a16e665a5cb5de21bd4d51bbbe0/datadog_checks_dev/datadog_checks/dev/kind.py#L46

Reply:
https://github.com/DataDog/integrations-core/pull/12939